recipes/robotraconteur/bld.bat

```diff
@@ -0,0 +1,20 @@
+@echo off
+
+md build2
+cd build2
+if %errorlevel% neq 0 exit /b %errorlevel%
+cmake -GNinja -DBUILD_TESTING=OFF -DBUILD_DOCUMENTATION=OFF -DBUILD_GEN=ON -DBUILD_PYTHON3=ON ^
+  -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
```

Comment:
```suggestion
   %CMAKE_ARGS% ^
```

Reply:
Can you show exactly what cmake args you require?

Reply:
If you set "echo on" in your script you will see what the variable expands to.

Reply:
Done